otherlibs/unix/unix.mli

```diff
@@ -865,7 +873,9 @@ val create_process :
    and causes the new process to have the same standard output
    as the current process.
    The executable file [prog] is searched in the path.
-   The new process has the same environment as the current process. *)
+   The new process has the same environment as the current process. 
+   The first element of the array [args] may or may not be required to
```

Comment:
`may or may not be required` sounds mostly confusing.

Reply:
I was vague because oy your comments on the same issue... I will rephrase as you suggest, but do you have a system in mind that does not follow the convention of first args being the process name ?

Maybe what is not clear is what happens when the first args is not the same as the process name.

In fact, my main concern is that if you do

Unix.open_process_args "cmd" [| "-l" |] it will not work as expected while 
Unix.open_process_args "cmd" [| ""; "-l" |] probably will (I did not test), because most commands look for option from argv[1] ?

Reply:
What I have in mind is that the specification of `program name` may vary. Typically, the C17 standard reads as
```
If the value of argc is greater than zero, the string pointed to by argv[0]
represents the program name; argv[0][0] shall be the null character if the
program name is not available from the host environment.
```
but the `represent` verb is quite ambiguous.

Famously, the busybox executable mimics the behavior of the program which name in its `argv[0]`. 

Overall, we should focus on the important points: we want to warn users that `argv[0]` has a special meaning.